src/Shell/Task/SeedTask.php

```diff
@@ -140,8 +164,92 @@ public function getOptionParser()
             'choices' => $bakeThemes
         ])->addArgument('name', [
             'help' => 'Name of the seed to bake. Can use Plugin.name to bake plugin models.'
+        ])->addOption('data', [
+            'short' => 'd',
+            'boolean' => true,
+            'help' => 'Include data from the table to the seed'
+        ])->addOption('fields', [
+            'short' => 'f',
```

Comment:
`f` is already used for `force` I suggest having no short form for this and also remove it for `--data`